src/components/feedback/Dialog.tsx

```diff
@@ -100,7 +100,7 @@ export default function Dialog({
   const {
     buttons,
     icon,
-    paddingSize = 'md',
+    paddingSize = 'lg',
```

Comment:
Let's make sure this does not become a breaking change. For that we need to check every place where `Dialog` or `ModalDialog` is used in client, lms and via, and make sure the new style is what we want.

Otherwise, let's keep the existing default value, and change it in those places where we actually want to use a large padding.